stormpath/resources/base.py

```diff
@@ -21,6 +22,27 @@
 SIGNAL_RESOURCE_DELETED = 'resource-deleted'
 
 
+class ResourceEncoder(JSONEncoder):
+    def default(self, o):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [An attribute defined in json.encoder line 159 hides this method](https://www.codacy.com/app/r/stormpath-sdk-python/file/2401272799/issues/source?bid=3181420&fileBranchId=3232321#l26)